client/components/seo/reader-preview/index.js

```diff
@@ -6,64 +6,30 @@ import React, { PropTypes, PureComponent } from 'react';
 /**
  * Internal dependencies
  */
-import humanDate from 'lib/human-date';
+import DisplayTypes from 'state/reader/posts/display-types';
+import ReaderPostCard from 'blocks/reader-post-card';
 
 export class ReaderPreview extends PureComponent {
 	render() {
 		const {
-			siteTitle,
-			siteSlug,
-			siteIcon,
-			postTitle,
+			site,
+			post,
 			postExcerpt,
 			postImage,
-			postDate,
-			authorName,
-			authorIcon
 		} = this.props;
 
+		const readerPost = { ...post };
+		if ( postImage ) {
+			readerPost.canonical_media = { src: postImage };
+		}
+		readerPost.better_excerpt = postExcerpt;
+
+		if ( postImage && ! postExcerpt ) {
+			readerPost.display_type = DisplayTypes.PHOTO_ONLY;
+		}
+
```

Comment:
Noice! I also figured out how to add an additional property to a sub-object in f305999eeaf4f1a72542270f306b0701c0f3eded. Let me know if I did it right 🗡